src/screens/EPub/components/EPubHeader/EPubToolbar.js

```diff
@@ -58,7 +58,7 @@ function EPubToolbar({ view, dispatch, epub }) {
     <CTFragment id="ct-epb-header-toolbar" justConBetween>
       <CTFragment alignItCenter className="ct-epb tool-btns">
         {saveBtnEl}
-        {previewBtnEl}
+        {null && previewBtnEl}
```

Comment:
Add a code comment, something like "Don't show preview button. The button causes a crash when clicked" (cause unknown). Creating a gh issue that references this code /the button code might be nice too.

Reply:
Done!